nano/rpc_test/rpc.cpp

```diff
@@ -6870,14 +6870,38 @@ TEST (rpc, confirmation_info)
 TEST (rpc, election_statistics)
 {
 	nano::test::system system;
-	auto node1 = add_ipc_enabled_node (system);
+	nano::node_config node_config;
+	node_config.ipc_config.transport_tcp.enabled = true;
+	node_config.ipc_config.transport_tcp.port = system.get_available_port ();
+	nano::node_flags node_flags;
+	node_flags.disable_request_loop = true;
+	auto node1 (system.add_node (node_config, node_flags));
 	auto const rpc_ctx = add_rpc (system, node1);
-	boost::property_tree::ptree request1;
-	request1.put ("action", "election_statistics");
-	auto response1 (wait_response (system, rpc_ctx, request1));
-	ASSERT_EQ ("0", response1.get<std::string> ("normal"));
-	ASSERT_EQ ("0", response1.get<std::string> ("hinted"));
-	ASSERT_EQ ("0", response1.get<std::string> ("optimistic"));
-	ASSERT_EQ ("0", response1.get<std::string> ("total"));
-	ASSERT_EQ ("0.00", response1.get<std::string> ("aec_utilization_percentage"));
+
+	nano::block_builder builder;
+	auto send1 = builder
+				 .send ()
+				 .previous (nano::dev::genesis->hash ())
+				 .destination (nano::public_key ())
+				 .balance (nano::dev::constants.genesis_amount - 100)
+				 .sign (nano::dev::genesis_key.prv, nano::dev::genesis_key.pub)
+				 .work (*system.work.generate (nano::dev::genesis->hash ()))
+				 .build ();
+	node1->process_active (send1);
+	ASSERT_TIMELY (5s, !node1->active.empty ());
```

Comment:
This line looks strange to me. Maybe there is a race condition between process_active() and start_elections(). The usual way to use start_elections() is:
```
	ASSERT_TRUE (nano::test::process (*node1, {send1}));
	ASSERT_TRUE (nano::test::start_elections (system, *node1, {send1}));
```